_posts/2025-01-30-socratic-seminar-31.md

```diff
@@ -23,64 +23,50 @@ meetup: https://www.meetup.com/pt-BR/clubebitcoinsp/events/305867752/
-    - [OP_NEXT](https://blockspace.media/opnext-bitcoin-scaling/)
+* [Tabconf 2024](https://www.youtube.com/@tabconf/videos) and [OP_NEXT](https://blockspace.media/opnext-bitcoin-scaling/) Recordings
 * [CTB6 codebase](https://github.com/CasaVinteUm/ctb6)
-* [Mempool bugado](https://x.com/0xB10C/status/1869781703646146706)
 * [jamesob - BITCOIN CORE'S LOSS OF FOCUS](https://x.com/jamesob/status/1860340932706730261)
-* [Deanonymization attacks against centralized coinjoin](https://bitcoinops.org/en/newsletters/2025/01/03/#deanonymization-attacks-against-centralized-coinjoin)
-    - [bitcoinmagazine.com](https://bitcoinmagazine.com/technical/wabisabi-deanonymization-vulnerability-disclosed)
-* [ATH hashrate jan 2 2025, a lot of blocks in a very short period of time:](https://bitcoinisdata.com/hash_rate/)
-    - [mempool.space](https://mempool.space/graphs/mining/hashrate-difficulty)
+* [Deanonymization attacks against centralized coinjoin](https://bitcoinmagazine.com/technical/wabisabi-deanonymization-vulnerability-disclosed)
 * [Fifteen OFAC-sanctioned transactions missing from blocks](https://b10c.me/observations/13-missing-sanctioned-transactions-2024-12/)
 * [Burak did it again, without breaking anything this time](https://x.com/brqgoo/status/1881768183314657751)
 
 ### Covenants
 
 * [Covenants support table:](https://en.bitcoin.it/wiki/Covenants_support)
 * [CTV, APO, CAT activity on signet](https://delvingbitcoin.org/t/ctv-apo-cat-activity-on-signet/1257)
-* [Un-FE’d Covenants: Char-ting a new path to Emulated Covenants via BitVM Integrity Checks](https://rubin.io/public/pdfs/unfedcovenants.pdf)
 * [P2A + 1p1c package relay + CTV na signet](https://x.com/1440000bytes/status/1868120931366817932)
-* [CTV + Pay To Anchor Example](https://github.com/stutxo/simple_ctv)
+ - [Code](https://github.com/stutxo/simple_ctv)
 
 ### Bitcoin
 
+* [Nunchuck taproot multisig](https://www.nobsbitcoin.com/nunchuk-launches-taproot-multisig-wallets-beta-on-mobile/)
 * [Zero fee txs + P2A address](https://x.com/jaoNoctus/status/1867838814032625981)
 * [O que está acontecendo na testnet4](https://blog.dlsouza.lol/bitcoin/testnet/2024/12/02/testnet4.html)
-* [Basic Bitcoin Lisp Language (bll)](https://github.com/ajtowns/bllsh)
 * [Bitcoin Output Script Descriptor (BOSD) to address issues of expressiveness, succinctness, and correctness when specifying Bitcoin transaction output types and standardness rules, particularly for L2 withdrawals.](https://github.com/alpenlabs/bitcoin-bosd)
 
 ### Crypto(graphy)
 
 * [NUMS (Nothing Up My Sleeve) keys](https://delvingbitcoin.org/t/unspendable-keys-in-descriptors/304/32)
 * [@rusty_twit's Great Script Restoration (GSR) offers a potential path toward post-quantum secure Bitcoin](https://x.com/n1ckler/status/1854552545084977320)
 * [ZCash FROST repo now supports secp256k1 curve on top of taproot (BIP 340 and 341 compatible)](https://github.com/ZcashFoundation/frost/pull/730)
```

Comment:
Is anyone actually going to talk about FROST?

Reply:
Botanix Labs is using this, and also Alpen might use for some internal stuff as well (unclear now).

Reply:
if you're going to talk about it, then keep it

![image](https://github.com/user-attachments/assets/f2f3164c-ee28-4dc0-a152-b902ef845281)

Reply:
That depends... I am josediurnus...